README.md

```diff
@@ -1,2 +1,35 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+Flowchart : https://i.gyazo.com/4f236f8ca944556c4a966e149faec5b7.png
+
+points = 0
+secretWord = "Yeet"
```

Comment:
You have hard-coded the secret word here -- better to not do that, so that your algorithm can work for any secret word